parm/config/config.fcst

```diff
@@ -255,14 +325,17 @@ elif [[ "$CDUMP" == "gfs" ]] ; then # GFS cycle specific parameters
       export io_layout="1,1"
     fi
 
-
 fi
 
-    ## Debug load balancing
-    # export KEEPDATA="YES"
-    # export ESMF_RUNTIME_PROFILE=ON
-    # export ESMF_RUNTIME_PROFILE_OUTPUT=SUMMARY
-
+if [ $cplgocart = .true. ]; then # temporary settings for aerosol coupling
+    export DIAG_TABLE="$HOMEgfs/parm/parm_fv3diag/diag_table_aer"
+    export FIELD_TABLE="$HOMEgfs/parm/parm_fv3diag/field_table_aer"
+    export CHM_CFGDIR="not_set"
+    export CHM_INPDIR="not_set"
+    export dnats="$(( $dnats + 0 ))"
+elif [[ $cpl == .true. ]] ; then # coupled model
```

Comment:
Same here, do we need a DO_COUPLED variable?

Reply:
Changed to `elif [[ $DO_COUPLED == "YES" ]] ; then # coupled model`